pkg/gce-pd-csi-driver/utils.go

```diff
@@ -255,6 +262,16 @@ func CodeForError(err error) *codes.Code {
 	return &internalErrorCode
 }
 
+func existingErrorCode(err error) *codes.Code {
+	if err == nil {
+		return nil
+	}
+	if status, ok := status.FromError(err); ok {
+		return errCodePtr(status.Code())
```

Comment:
Since we are returning from multiple places, should we add more logs? For example on line 239, 249, 262 can return internal error code but we do not know exactly which one is returning it.

Reply:
I don't really think it matters if the internal error is returned on 239, 249, or 262. That being said, if we really want to know what the error was/why internal error was returned, `LoggedError` should have already logged what the error was before entering the `CodeForError` function, so I think adding logging within here is a little excessive and unnecessary. What do you think?